<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountlist</class>
 <widget class="QMainWindow" name="accountlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>帳號</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>161</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>false</underline>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>47</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>帳號列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>選擇</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
